src/deploy/standalone/logrotate_noobaa.sh

```diff
@@ -0,0 +1 @@
+/usr/sbin/logrotate -v /etc/logrotate.d/noobaa/logrotate_noobaa.conf
```

Comment:
add hasbang so that you don't need to specify bash to call it:
```suggestion
#!/bin/bash
/usr/sbin/logrotate -v /etc/logrotate.d/noobaa/logrotate_noobaa.conf
```

Reply:
@guymguym  I tried with hasbang but script is not getting executed 
error
`program '/usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.sh' (pid 518) exited with status 1 [v8.2310.0-4.el9]
May 06 12:51:14 d310af06fbec rsyslogd[512]: file size limit cmd for file '/var/log/noobaa.log' did no resolve situation  [v8.2310.0-4.el9] `

Reply:
so add `|| true` at the end...

Reply:
outchannel is giving issues when script is not executable

Reply:
chmod +x?
Also I think the channel size should also be a bit higher than the logrotate max-size to assure that logrotate will not skip it and cause rsyslog to stop writing.